<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogPacient</class>
 <widget class="QDialog" name="DialogPacient">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>415</width>
    <height>394</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Nový pacient</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="8" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Telefon 2:</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Poznámka k telefonu:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="textJmeno">
       <property name="styleSheet">
        <string notr="true">font-weight: bold;</string>
       </property>
       <property name="text">
        <string>Jméno:</string>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Poznámka k telefonu 2:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="textObec">
       <property name="styleSheet">
        <string notr="true">font-weight: bold;</string>
       </property>
       <property name="text">
        <string>PSČ, obec:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="textUlice">
       <property name="styleSheet">
        <string notr="true">font-weight: bold;</string>
       </property>
       <property name="text">
        <string>Ulice, číslo:</string>
       </property>
      </widget>
     </item>
     <item row="10" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Poznámka k pacientovi:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="textIdPacienta">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="textPrijmeni">
       <property name="toolTip">
        <string>Příjmení</string>
       </property>
       <property name="styleSheet">
        <string notr="true">font-weight: bold;</string>
       </property>
       <property name="text">
        <string>Příjmení:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="textTelefon">
       <property name="styleSheet">
        <string notr="true">font-weight: bold;</string>
       </property>
       <property name="text">
        <string>Telefon:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="vstupTelefon">
       <property name="toolTip">
        <string>Telefon</string>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Normal</enum>
       </property>
       <property name="placeholderText">
        <string>9 číslic bez mezer</string>
       </property>
      </widget>
     </item>
     <item row="6" column="2">
      <widget class="QCheckBox" name="volbaNemaTelefon">
       <property name="text">
        <string>nemá telefon</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="2">
      <widget class="QLineEdit" name="vstupPrijmeni">
       <property name="toolTip">
        <string>Příjmení</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1" colspan="2">
      <widget class="QLineEdit" name="vstupUlice">
       <property name="toolTip">
        <string>Ulice, číslo</string>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Normal</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="1" colspan="2">
      <widget class="QLineEdit" name="vstupJmeno">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="toolTip">
        <string>Jméno</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="2">
      <widget class="QLabel" name="popisekIdPacienta">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="5" column="1" colspan="2">
      <widget class="QLineEdit" name="vstupObec">
       <property name="toolTip">
        <string>PSČ, obec</string>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Normal</enum>
       </property>
      </widget>
     </item>
     <item row="7" column="1" colspan="2">
      <widget class="QLineEdit" name="vstupPoznTel">
       <property name="toolTip">
        <string>Poznámka k telefonu</string>
       </property>
       <property name="placeholderText">
        <string>př. &quot;dcera&quot;, &quot;manžel&quot; apod.</string>
       </property>
      </widget>
     </item>
     <item row="10" column="1" colspan="2">
      <widget class="QLineEdit" name="vstupPoznamka">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="toolTip">
        <string>Poznámka k pacientovi</string>
       </property>
       <property name="placeholderText">
        <string>př. kde nechat zboží v nepřítomnosti</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1" colspan="2">
      <widget class="QLineEdit" name="vstupPoznTel2">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="toolTip">
        <string>Poznámka k telefonu 2</string>
       </property>
       <property name="placeholderText">
        <string/>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QLineEdit" name="vstupTelefon2">
       <property name="toolTip">
        <string>Telefon 2</string>
       </property>
       <property name="placeholderText">
        <string>9 číslic bez mezer</string>
       </property>
      </widget>
     </item>
     <item row="11" column="0" colspan="3">
      <widget class="QLabel" name="label_7">
       <property name="styleSheet">
        <string notr="true">font-weight: bold;</string>
       </property>
       <property name="text">
        <string>Tučně vyznačené položky jsou povinné</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="1" colspan="2">
      <widget class="QCheckBox" name="volbaUpravitJmeno">
       <property name="text">
        <string>upravit jméno nebo příjmení</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="popisekUpozorneni">
     <property name="styleSheet">
      <string notr="true">color: red;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="tlacitkoZrusit">
       <property name="toolTip">
        <string>Zrušit (Esc)</string>
       </property>
       <property name="text">
        <string>Zrušit</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="tlacitkoUlozit">
       <property name="toolTip">
        <string>Uložit (Ctrl+Enter)</string>
       </property>
       <property name="text">
        <string>Uložit</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
       <property name="default">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>vstupPrijmeni</tabstop>
  <tabstop>vstupJmeno</tabstop>
  <tabstop>vstupUlice</tabstop>
  <tabstop>vstupObec</tabstop>
  <tabstop>vstupTelefon</tabstop>
  <tabstop>volbaNemaTelefon</tabstop>
  <tabstop>vstupPoznTel</tabstop>
  <tabstop>vstupTelefon2</tabstop>
  <tabstop>vstupPoznTel2</tabstop>
  <tabstop>vstupPoznamka</tabstop>
  <tabstop>tlacitkoZrusit</tabstop>
  <tabstop>tlacitkoUlozit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
